### PR TITLE
Bf 419 - Prepare script(job) to change jira statuses after BF Env update

### PR DESCRIPTION
**Pull request by Mykhailo-Teliatnikov**
Create jenkins pipeline for change status Jira tikers depend on service version that was deployed to Bugfest 
Modify JiraIssue and JiraParser.JiraIssue  classes

Jenkins job: https://jenkins-aws.indexdata.com/job/Rancher/job/BF-419/

### CODE REVIEW

File: pipelines/automation/jira-tickers-move.groovy [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+                else {
+                    println "Error in input data"
+                }
+            }
+            stage('Search Jira tasks') {
+                List<JiraIssue> issues = jiraClient.searchIssues(search_pattern,
+                    ["key","status","fixVersions","project"])
+                list_of_found_jira_tasks=SortJiraTickersByVersion(issues)
+            }
+            stage('Get map with services from bugfes') {
```
Comment (vitalii-ustalu) [inline pipelines/automation/jira-tickers-move.groovy:67]: bugfest*

File: pipelines/automation/jira-tickers-move.groovy [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+                        }
+                        if (check_is_less) {
+                            list_of_jira_tasks_to_change.add(i)
+                        }
+                    }catch (exception){
+                        println(exception)
+                    }
+                }
+            }
+            stage('Change status of Jira ticker') {
```
Comment (vitalii-ustalu) [inline pipelines/automation/jira-tickers-move.groovy:115]: ticket*

Comment (Mykhailo-Teliatnikov) [reply pipelines/automation/jira-tickers-move.groovy]: Changed

File: pipelines/automation/jira-tickers-move.groovy (hunk 1 above)
Comment (Mykhailo-Teliatnikov) [reply pipelines/automation/jira-tickers-move.groovy]: Changed

File: pipelines/automation/jira-tickers-move.groovy [hunk 3]
```diff
@@ -0,0 +1,164 @@
+#!groovy
+@Library('pipelines-shared-library@BF-419') _
```
Comment (kdubniak) [inline pipelines/automation/jira-tickers-move.groovy:2]: Don`t forget delete before merging BF-419

**Review by kdubniak — APPROVED** [review]

**Review by renatsafiulin — APPROVED** [review]

**Review by vitalii-ustalu — APPROVED** [review]

**Pull request merged**